src/core/namespace.c

```diff
@@ -1374,6 +1393,60 @@ static bool home_read_only(
         return false;
 }
 
+static int verity_settings_prepare(
+                VeritySettings *verity,
```

Comment:
ret_verity? Although it's not allocated here, it's modified

Reply:
the ret_ prefix we use for actual return parameters, i.e. stuff that we *initialize* on success on don't touch on failure. This is different: we get something passed in that already exists, initialized, configured, we just do some additional operation on it. It's not a return parameter, it's both input *and* output, and hence it should not carry ret_.

(We have not separate prefix for stuff that is only input or both input and output, we only have the ret_ prefix for stuff that is only output...)

Reply:
ok makes sense